packages/framework/build.xml

```diff
@@ -1160,4 +1161,18 @@
         <property name="yaml-standards.args" value="${yaml-standards.args} --fix" override="true"/>
         <phingcall target="yaml-standards"/>
     </target>
+
+    <target name="stylelint" description="Run style linter.">
+        <exec executable="${path.root}/${path.stylelint.executable}" logoutput="true" passthru="true" checkreturn="true" dir="${path.root}">
+            <arg value="--color"/>
+            <arg path="${path.src}/**/*.less"/>
+        </exec>
+    </target>
+
+    <target name="stylelint-fix" description="Run CSS linter.">
+        <exec executable="${path.root}/${path.stylelint.executable}" logoutput="true" passthru="true" checkreturn="true" dir="${path.root}">
+            <arg path="${path.src}/**/*.less"/>
+            <arg value="--fix"/>
+        </exec>
+    </target>
```

Comment:
Please move both stylelint targets after `standards-fix-diff` as they should be sorted alphabetically and `phing-config-check` fails on this

Reply:
Yes, I saw it in travis. I didn't know that we have rule for alphabetically sort phing targets :smile: